ctx.go

```diff
@@ -307,6 +346,11 @@ func (c ContextNoBody) QueryParamInt(name string, defaultValue int) int {
 func (c ContextNoBody) QueryParamBoolErr(name string) (bool, error) {
```

Comment:
> // If the query parameter does not exist or is not a bool, it returns nil.

It seems this always returns a bool as the first return type? Maybe look at this comment while we're here.